packages/safe-core-sdk/src/ethereumLibs/EthersAdapter.ts

```diff
@@ -151,6 +151,14 @@ class EthersAdapter implements EthAdapter {
   call(transaction: EthAdapterTransaction): Promise<string> {
     return this.#provider.call(transaction)
   }
+
+  ensLookup(name: string): Promise<string | null> {
```

Comment:
Are you defining the return type as `string | null` because you expect it to throw? Or does it actually return `null` when the name is not found? If it's the former, you need not specify the `null`, e.g. `Promise<string>`

This goes for `ensReverseLookup` below as well.

Reply:
The Ethers type definitions we use specify `Promise<string | null>` as return value for both lookups so we had to change it here. The [official documentation](https://docs.ethers.io/v5/single-page/#/v5/api/providers/provider/-%23-Provider-lookupAddress) only has `Promise<string>` defined though.

Reply:
Alright, I'll resolve this, but can you add it to the issue we created the other day about updating the types?